.circleci/config.yml

```diff
@@ -157,6 +157,104 @@ references:
         python3 --version
         demisto-sdk --version
 
+  download_configuration: &download_configuration
+    run:
+      name: Download Configuration
+      when: always
+      command: |
+        if ./Tests/scripts/is_ami.sh ;
+          then
+            ./Tests/scripts/download_demisto_conf.sh
+
+          else
+            ./Tests/lastest_server_build_scripts/download_demisto_conf.sh
+        fi
+  
+  test_download_upload: &test_download_upload
+    run:
+      name: Test Download and Upload commands
+      when: on_success
+      command: |
+        echo "Running upload"
+        demisto-sdk upload -i Packs/HelloWorld/Integrations/ --insecure > artifacts/upload.log
+
+        echo "Running download"
+        demisto-sdk download -o Packs/HelloWorld/ -i HelloWorld --insecure -f > artifacts/download.log
+    
+        echo "Running format"
+        { # try
+            yes | demisto-sdk format -i Packs/HelloWorld/Integrations/HelloWorld -nv -ud > artifacts/format.log
+        } || { # catch 
+            echo "Format exit status: $?"  # DO NOT DELETE THIS LINE. format sometimes will exit with the odd exist statuses.
+        }
+        echo "Running validate"
+        demisto-sdk validate -i Packs/HelloWorld/Integrations/HelloWorld/ > artifacts/validate.log
+        if [[ $? -ne 0 ]]
+        then
+            VALDIATE_CODE=1
+        fi
+
+        echo "Running lint"
+        demisto-sdk lint -i Packs/HelloWorld/Integrations/HelloWorld/ > artifacts/lint.log
+        if [[ $? -ne 0 ]]
+        then
+            LINT_CODE=1
+        fi
+        
+        # Check codes
```

Comment:
1) why aren't you checking the status code for download and upload?
2) currently you ignore the status of format, so why run it?
3) why do all the commands run on the integration and not the pack?

Reply:
When download or upload are going back, they'll fail the task if not 1. 
I'm ignoring the format outputs as some of its threads will fail on files in TestData.
I'll try to run on the pack. I can't run upload on the pack as it will fail (our server is going up with one helloworld pack).